index.html

```diff
@@ -0,0 +1,408 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Smashing Magazine Website Heat Map</title>
+  <link rel="stylesheet" href="./css/styles.css">
+</head>
+<body>
+  <header class="header">
+    <div class="header-left">
+      <div class="header-logo"></div>
+      <div class="header-logo-name">
+        <div></div>
+        <div></div>
+      </div>
+    </div>
+    <div class="header-middle">
+      <div class="nav-item"></div>
+      <div class="nav-item"></div>
+      <div class="nav-item"></div>
+      <div class="nav-item"></div>
+      <div class="nav-item"></div>
+      <div class="nav-item"></div>
+      <div class="nav-item">
+        <div></div>
+        <div></div>
+      </div>
+  </div>
+
+  <div class="header-right">
+    <input type="search" name="article" >
+  </div>
+    
+  </header>
```

Comment:
- [ ] [OPTIONAL] The navigation bar should use nav element with ul/li inside, however, you are allowed to used div as the main focus is a heat map. (content is not important)